test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -262,21 +262,23 @@ public void Constructor_SetsContentTypeAndParameters()
         {
             // Arrange
             // Point the IFileProvider root to a different subfolder
-            var fileProvider = new PhysicalFileProvider(Path.GetFullPath("./Properties"));
-            var filePathResult = new VirtualFileResult(path, "text/plain")
+            using (var fileProvider = new PhysicalFileProvider(Path.GetFullPath("./Properties")))
```

Comment:
We might be able to do away with not using a `PhysicalFileProvider` here. It looks like all it cares about is to get a `NotFound` file result which we could mock rather than use a real file.